Kernel/CommandLine.cpp

```diff
@@ -59,14 +59,19 @@ UNMAP_AFTER_INIT void CommandLine::add_arguments(Vector<StringView> const& args)
             continue;
         }
 
-        auto pair = str.split_view('=');
-        VERIFY(pair.size() == 2 || pair.size() == 1);
-
-        if (pair.size() == 1) {
-            m_params.set(pair[0], ""sv);
-        } else {
-            m_params.set(pair[0], pair[1]);
-        }
+        // Note: The kernel command line might include entries like
+        // ```
+        // video=HDMI-A-1:1920x1080M@30D,margin_left=48,margin_right=48,margin_top=48,margin_bottom=48
+        // ```
+        // which is one entry `video` with comma separated list of values
+        // we just represent that as the string of all values for now and
+        // let a later parser handle that
+        // TLDR: Values might contain '='
```

Comment:
This comment is a bit verbose. We also don't normally include markdown block quotes in our comments.

Maybe something like:

```
// Some boot loaders may include complex key-value pairs where the value is a composite entry.
// Handle this by only checking for the first equals sign in each command line parameter
```

And then you could move the example to the commit?